tests/Makefile

```diff
@@ -94,6 +94,8 @@ slow-query: currdir
 tck: jobs slow-query
 	$(test_j) tck/steps/test_tck.py
 
+test-all: test tck
```

Comment:
Currently using `make tck` to start test cases. why support another command?

Reply:
Just for typing less.

Reply:
fine  it's an alias